packages/ember-debug/lib/error.js

```diff
@@ -18,15 +7,12 @@ function ExtendBuiltin(klass) {
   @constructor
   @public
 */
-export default class EmberError extends ExtendBuiltin(Error) {
-  constructor(message) {
-    super();
 
-    if (!(this instanceof EmberError)) {
-      return new EmberError(message);
-    }
+export default EmberError;
+function EmberError(message, code) {
```

Comment:
Dumb question, but why make this a function instead of using the fancy ES6 classes?